easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.6.0-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb

```diff
@@ -0,0 +1,41 @@
+name = 'TensorFlow'
+version = '1.6.0'
+cuda_ver = '9.1.85'
+versionsuffix = '-Python-%%(pyver)s-CUDA-%s' % cuda_ver
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-%(version)s_swig-env.patch',
```

Comment:
@smoors Why not use the existing `TensorFlow-1.5.0_swig-env.patch` which applies just fine for TF 1.6.0?